browser-test/src/support/admin_programs.ts

```diff
@@ -78,6 +78,11 @@ export class AdminPrograms {
     await waitForPageJsLoad(this.page)
   }
 
+  async gotoDisabledProgramIndexPage() {
```

Comment:
**suggestion, non-blocking**: this is assuming a starting place (the program index page?) is that the behavior the other `goto...` methods on this object have, or should we navigate to the index page before clicking the tab?